### ISSUE TITLE
fine-tune pre-trained(sports-1m) C3D model with my own dataset

### ISSUE THREAD

**Issue by deepaksharma36** [facebook/C3D#200]
How I can fine-tune pre-trained(sports-1m) C3D model with my own dataset.

**Comment by deepaksharma36** [comment]
I just made changes in train01.lst for pointing the paths to my dataset and run the ucf101_finetuning.sh script after running the create_volume_mean.sh script. Thats it or I need to make some other changes?

**Comment by dutran** [comment]
@deepaksharma36 you need to
1) prepare your list files (both train and test)
2) create a volume mean - using train list
3) fine tune (remember to change prototxt file both for the network, e.g. different fc layer with different names, as your prediction layer may have different dimension, change different loss, if your loss layer is different)
4) use finetuning tool to fine to and use test net to test, or extract image features to extract features, predictions.
